trace_statetests_new.py

```
@@ -324,13 +313,14 @@ def generateTests():
 # to resume running after interruptions
 START_I = 0
 
-
-
-def randomTestIterator():
+def iterate_tests():
+    test_generator = generateTests
```

I would prefer it if you default to random tests.. I'm running this on a cloud machine, and if I merge this as is, it will break that instance